<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMVxClass</class>
 <widget class="QMainWindow" name="WMVxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WMVx</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="RenderWidget" name="renderWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuScene">
     <property name="title">
      <string>Scene</string>
     </property>
     <addaction name="actionSceneLoad"/>
     <addaction name="actionSceneSave"/>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="actionLoad_Client"/>
    <addaction name="actionUpdate_Support_Files"/>
    <addaction name="actionView_Log"/>
    <addaction name="separator"/>
    <addaction name="menuScene"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Image"/>
    <addaction name="actionExport_Video"/>
    <addaction name="actionExport_3D"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Layout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuCamera">
     <property name="title">
      <string>Camera</string>
     </property>
     <addaction name="actionCamera_Reset"/>
    </widget>
    <widget class="QMenu" name="menuBackground">
     <property name="title">
      <string>Background</string>
     </property>
     <addaction name="actionBGColor"/>
     <addaction name="actionBGTransparent"/>
    </widget>
    <addaction name="actionShow_Library_Panel"/>
    <addaction name="actionShow_Animation_Panel"/>
    <addaction name="actionShow_Scene_Panel"/>
    <addaction name="actionShow_Characters_Panel"/>
    <addaction name="actionShow_Model_Panel"/>
    <addaction name="actionShow_Render_Panel"/>
    <addaction name="actionShow_Log_Panel"/>
    <addaction name="separator"/>
    <addaction name="menuBackground"/>
    <addaction name="actionToggle_Grid"/>
    <addaction name="separator"/>
    <addaction name="menuCamera"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Dev_Tools"/>
    <addaction name="actionOpen_Texture_Tool"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_WMVx"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockLibrary">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabFiles">
        <attribute name="title">
         <string>Files</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="LibraryFilesControl" name="libraryFilesControl" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabNpcs">
        <attribute name="title">
         <string>NPCs</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="LibraryNpcsControl" name="libraryNpcsControl" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockModel">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Model</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Model</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
         <item>
          <widget class="ModelControl" name="modelControl" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Character</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CharacterControl" name="characterControl" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockAnimation">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Animation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="AnimationControl" name="animationControl" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockLog">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Live Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="plainTextLog">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
       <property name="maximumBlockCount">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockScene">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Scene</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="SceneControl" name="sceneControl" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockRender">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Render</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="RenderControl" name="renderControl" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionGame_Clients">
   <property name="text">
    <string>Game Clients</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionView_Log">
   <property name="text">
    <string>View Log</string>
   </property>
  </action>
  <action name="actionExport_Image">
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
  <action name="actionExport_Video">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Video</string>
   </property>
  </action>
  <action name="actionExport_3D">
   <property name="text">
    <string>Export 3D</string>
   </property>
  </action>
  <action name="actionReset_Layout">
   <property name="text">
    <string>Reset Layout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionView_NPC">
   <property name="text">
    <string>View NPC</string>
   </property>
  </action>
  <action name="actionView_Item">
   <property name="text">
    <string>View Item</string>
   </property>
  </action>
  <action name="actionShow_Library_Panel">
   <property name="text">
    <string>Show Library Panel</string>
   </property>
  </action>
  <action name="actionShow_Model_Panel">
   <property name="text">
    <string>Show Model Panel</string>
   </property>
  </action>
  <action name="actionShow_Scene_Panel">
   <property name="text">
    <string>Show Scene Panel</string>
   </property>
  </action>
  <action name="actionShow_Animation_Panel">
   <property name="text">
    <string>Show Animation Panel</string>
   </property>
  </action>
  <action name="actionToggle_Grid">
   <property name="text">
    <string>Toggle Grid</string>
   </property>
  </action>
  <action name="actionCamera_Reset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionleft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionLoad_Character">
   <property name="text">
    <string>Load Character</string>
   </property>
  </action>
  <action name="actionSave_Character">
   <property name="text">
    <string>Save Character</string>
   </property>
  </action>
  <action name="actionLoad_Equipment">
   <property name="text">
    <string>Load Equipment</string>
   </property>
  </action>
  <action name="actionSave_Equipment">
   <property name="text">
    <string>Save Equipment</string>
   </property>
  </action>
  <action name="actionClear_Equipment">
   <property name="text">
    <string>Clear Equipment</string>
   </property>
  </action>
  <action name="actionToggle_Mount">
   <property name="text">
    <string>Toggle Mount</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_WMVx">
   <property name="text">
    <string>About WMVx</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionOpen_Dev_Tools">
   <property name="text">
    <string>Open Dev Tools</string>
   </property>
  </action>
  <action name="actionSceneLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSceneSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionShow_Log_Panel">
   <property name="text">
    <string>Show Log Panel</string>
   </property>
  </action>
  <action name="actionLoad_Client">
   <property name="text">
    <string>Load Client</string>
   </property>
  </action>
  <action name="actionOpen_Texture_Tool">
   <property name="text">
    <string>Open Texture Tool</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBGColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBGTransparent">
   <property name="text">
    <string>Transparent</string>
   </property>
  </action>
  <action name="actionShow_Render_Panel">
   <property name="text">
    <string>Show Render Panel</string>
   </property>
  </action>
  <action name="actionUpdate_Support_Files">
   <property name="text">
    <string>Update Support Files</string>
   </property>
  </action>
  <zorder>dockRender</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LibraryFilesControl</class>
   <extends>QWidget</extends>
   <header>libraryfilescontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharacterControl</class>
   <extends>QWidget</extends>
   <header>charactercontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModelControl</class>
   <extends>QWidget</extends>
   <header>modelcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>renderwidget.h</header>
  </customwidget>
  <customwidget>
   <class>SceneControl</class>
   <extends>QWidget</extends>
   <header>scenecontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnimationControl</class>
   <extends>QWidget</extends>
   <header>animationcontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryNpcsControl</class>
   <extends>QWidget</extends>
   <header>librarynpcscontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RenderControl</class>
   <extends>QWidget</extends>
   <header>rendercontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="WMVx.qrc"/>
 </resources>
 <connections/>
</ui>
